growth/too/views.py

```diff
@@ -783,6 +783,18 @@ def plan_manual():
             telescope = form.telescope.data
             json_data, queue_name = get_json_data_manual(form)
 
+            mess = 'Target list empty. None of the requested fields ' + \
```

Comment:
* I find `msg` or `message` to be a clearer variable name than `mess`.
* The `+` is superfluous here; consecutive string literals are automatically concatenated.
* Consider being even more clear about what has failed: "No observations were scheduled because..."

Reply:
fixed.